src/material/legacy-core/theming/prebuilt/pink-bluegrey.scss

```diff
@@ -23,3 +24,5 @@ $theme: theming.define-dark-theme((
 
 // Include all theme styles for the components.
 @include all-theme.all-legacy-component-themes($theme);
+
+@include typography.typography-hierarchy($theme);
```

Comment:
We can land the changes as they are now, but for a follow-up PR: maybe we should bake this into the `core-typography` mixin?

Reply:
Yeah possibly - it seems weird to have to include this separately. I would have expected our theme files to be sufficient with an "all-component-themes" mixin.